tests/data/integration.json

```diff
@@ -0,0 +1,24 @@
+{
+  "SVcalling.sample": "sample-test",
+  "SVcalling.outputDir": "/exports/sascstudent/cedrick",
```

Comment:
Integration tests should not have the outputDir specified in the inputs json. They won't be able to run on travis like this.

Reply:
Although, for now this is okay since these tests aren't being run on travis yet anyway.